docs/context/ref/TransactionResult.md

```diff
@@ -0,0 +1,57 @@
+---
+id: TransactionResult
+sidebar_label: TransactionResult
+title: TransactionResult
+hide_title: true
+---
+# `TransactionResult`
+
+The result of any given create, update or delete intent.
+
+## Type
+
+`fdc3.transactionResult`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/transactionresult.schema.json
+
+## Details
+
+| Property    | Type    | Required | Example Value     |
+|-------------|---------|----------|-------------------|
+| `type`      | string  | Yes      | `'fdc3.transactionResult'`  |
+| `status`      | string  | Yes       | `'Updated'`      |
+| `context`  | string  | Yes       | See Below |
```

Comment:
Type should be Context, which will allow any other FDC3 context to be wrapped

Reply:
This has been updated now

Reply:
Not seeing the change... however the type filed is now Context... will add a suggestion to correct both